### PR TITLE
https://issues.redhat.com/browse/ACM-6935 CIM OS image detail

### PR DESCRIPTION
**Pull request by oafischer**

### CODE REVIEW

**Review by nmagnezi — APPROVED** [review]
/lgtm

File: clusters/cluster_lifecycle/cim_intro.adoc [hunk 1]
```diff
@@ -12,13 +12,13 @@ A machine joins the host inventory after booting with a Discovery Image. The Dis
 - An agent that performs discovery, validation, and installation tasks.
 - The necessary configuration for reaching the service on the hub cluster, including the endpoint, token, and static network configuration, if applicable.
 
-You generally have a single Discovery Image for each infrastructure environment, which is a set of hosts sharing a common set of properties. The InfraEnv custom resource definition represents this infrastructure environment and associated Discovery Image.
+You generally have a single Discovery Image for each infrastructure environment, which is a set of hosts sharing a common set of properties. The `InfraEnv` custom resource definition represents this infrastructure environment and associated Discovery Image. The image used is based on your {ocp-short} version, which determines the OS version that is selected.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/cim_intro.adoc]:
```suggestion
You generally have a single Discovery Image for each infrastructure environment, which is a set of hosts sharing a common set of properties. The `InfraEnv` custom resource definition represents this infrastructure environment and associated Discovery Image. The image used is based on your {ocp-short} version, which determines the operating system version that is selected.
```

**Review by dockerymick — APPROVED** [review]
I assume that OS is operating system, if not please correct it haha. It is best to. write out OS. Otherwise, this looks good!

File: clusters/cluster_lifecycle/cim_intro.adoc (hunk 1 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/cim_intro.adoc]: Indeed, fixed, thanks!

**Pull request merged**